script/Ciel/Day 5/05_01_CIEL05_1D.txt

```diff
@@ -1,48 +1,54 @@
 [sha:d98b8590de1a88b7ebc7460819266e5a7ce03525]{
 -- Page 10004, Offset 11284.
 -- ……息を殺して様子を見る。
-...I stifle my breathing and take a look around me.
+I hold my breath and take stock of the situation.
 }
 [sha:1fbb05e896077e1921cc1d3267ee77bdf46a676c]{
 -- Page 10004, Offset 11285.
 -- 追いかけてくる足音は途絶えていた。
-// It's the sound of their footsteps actually
-I no longer hear the pursuing footsteps.
+I can't hear any footsteps coming after us anymore.
 }
 [sha:6a833ebfbd59acf4fe7d5314735679f0a5be5f9c]{
 -- Page 148, Offset 11286.
 -- あの死者たちの動きは人間なみだが、終始素早い訳じゃない。基本的には緩慢だ。
-The movements of the Dead are human-like, but it's not like they're on alert the entire time. Basically, they're sluggish.
+Those Dead may move like humans, but they aren't able to keep it up for long.
+ They're fundamentally sluggish by nature.
 }
 [sha:d4318ca06b78287c19036ca463721f6a66a380dc]{
 -- Page 148, Offset 11287.
 -- やつらは獲物を知覚した時だけ活動的になる。その範囲もそう広くない。せいぜい10メートル程だろう。
-They only move actively when they perceive their prey. Their range is not that wide, maybe ten meters at most.
+They only spring into action when they can perceive their prey,
+ and it seems like they can only sense things that are fairly close by.
+ Probably only ten meters or so at most.
 }
 [sha:70198b77071dae624924dd814aac8b54f349d16f]{
 -- Page 149, Offset 11288.
 -- ……知覚する方法が視力によるものなのか、聴覚によるものなのか、あるいは別の何かなのかは、アレになってみなければ判らない話ではあるが。
-...Whether they perceive by sight, hearing, or something else, I wouldn't know unless I turned into one.
+I don't know whether they perceive things based on sight, hearing, or something else entirely,
```

Comment:
i think it sounds better with "hearing" first
hearing, sight, or something else entirely.